src/linter.ts

```diff
@@ -88,13 +88,25 @@ export const lintNode = (
 
     results = results.concat(
       targetResults.map(result => {
-        const location = parsedResult.getLocationForJsonPath(parsed, result.path || targetPath, true);
+        const path = result.path || targetPath;
+        const location = parsedResult.getLocationForJsonPath(parsed, path, true);
 
         return {
           code: rule.name,
-          summary: rule.summary,
-          message: result.message,
-          path: result.path || targetPath,
+          get summary() {
```

Comment:
Nitpick: new line

Reply:
I guess this is a property because we can't use `this` if not in a defined function, right?